fiftyone/plugins/secrets.py

```diff
@@ -26,72 +29,68 @@ def __new__(cls):
             cls._instance.client = _get_secrets_client()
         return cls._instance
 
-    def register_operator(
-        self, operator_uri: str, required_secrets: typing.List[str]
-    ) -> None:
-        self._registered_secrets[operator_uri] = required_secrets
+    def register_operator(self, operator_uri, secrets):
+        self._registered_secrets[operator_uri] = secrets
 
-    def client(self) -> fois.ISecretProvider:
+    def client(self):
         if not self._instance:
             self._instance = self.__new__(self.__class__)
         return self._instance.client
 
-    async def get_secret(
-        self, key: str, operator_uri: str, **kwargs
-    ) -> Optional[fois.ISecret]:
-        """
-        Get the value of a secret.
+    async def get_secret(self, key, operator_uri, **kwargs):
```

Comment:
Why are we removing type annotations? This feels like it's just creating more tech debt.